ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -123,7 +124,13 @@ sub handle_service
         return;
     }
 
-    $self->prepare_action($srv, $file, $actions) if ($fh->close());
+    my $changes = $fh->close();
+    if ( $changes > 0 ) {
```

Comment:
i'm unaware that this could return a negative value. even if so, this should be handled in `CAF::FileWriter`, not in every component.